lib/mustache.mli

```diff
@@ -211,7 +225,7 @@ module With_locations : sig
   val section : loc:loc -> dotted_name -> t -> t
 
   (** [{{> box}}] *)
-  val partial : loc:loc -> name -> t
+  val partial : loc:loc -> int -> name -> t
```

Comment:
I think most users really don't care about that indentation. At least make it optional.